### PR TITLE
fix(tag): updating outline css to fix background

### PR DESCRIPTION
**Pull request by brianferry**
## What I did

1.  Updated the `rh-tag` stylesheet to fix the issues related to the `outline` variant styles not being applied for the background color.  

## Testing Instructions

1. Go to https://deploy-preview-1134--red-hat-design-system.netlify.app/elements/tag/demo/variants/
2. The tags should now all have the correct background color, text color, and stylings for the outline and filled variants.

### CODE REVIEW

**Review by bennypowers — CHANGES_REQUESTED** [review]
Needs a changeset describing the improvements to end user experience

File: .changeset/breezy-scissors-approve.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@rhds/elements": patch
+---
+
+`rh-tag`: outline variant should now get the correct background styles added.
```
Comment (bennypowers) [inline .changeset/breezy-scissors-approve.md:5]:
```suggestion
`<rh-tag>`: outline variant should now get the correct background styles added.
```

File: .changeset/breezy-scissors-approve.md [hunk 2]
```diff
@@ -2,4 +2,4 @@
 "@rhds/elements": patch
 ---
 
-`rh-tag`: outline variant should now get the correct background styles added.
+`<rh-tag>`: outline variant should now get the correct background styles added.
```
Comment (bennypowers) [inline .changeset/breezy-scissors-approve.md:5]: @hellogreg @coreyvickery is this a good release notes line item for Bellsprout? Needs any revisions? Would you be happy automatically pulling this in to the next round of release notes?

Comment (coreyvickery) [reply .changeset/breezy-scissors-approve.md:5]: @bennypowers Next round is fine.

**Review by bennypowers — APPROVED** [review]

**Pull request merged**